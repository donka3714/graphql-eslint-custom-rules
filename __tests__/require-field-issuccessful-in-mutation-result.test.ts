import { GraphQLRuleTester } from '@graphql-eslint/eslint-plugin'
import rule from '../src/rules/require-field-issuccessful-in-mutation-result';

const ruleTester = new GraphQLRuleTester()

ruleTester.runGraphQLTests('require-field-issuccessful-in-mutation-result', rule, {
  valid: [
    {
      code: 'query something { foo }'
    }
  ],
  invalid: [
    {
      code: 'query invalid { foo }',
      errors: [{ message: 'Your error message.' }]
    }
  ]
})