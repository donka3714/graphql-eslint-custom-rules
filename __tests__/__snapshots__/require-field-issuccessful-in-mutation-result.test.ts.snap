// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invalid #1 1`] = `
"#### ⌨️ Code

      1 | query invalid { foo }"
`;
